cea/tests/__init__.py

```diff
@@ -35,8 +38,22 @@ def main(config):
         # make sure we're working on a clean slate
         shutil.rmtree(default_config.project)
 
+    cov = coverage.Coverage()
```

Comment:
wait... you hardcoded coverage into the app? shouldn't it be possible to just do ` coverage run cea/tests/__init__.py --workflow slow`? (actually. I just tested it. it _is_ possible...)

Can we at least make this an option in the config? Coverage slows down the testing - we don't really want to do that all the time...

Reply:
Alright, in that case, I think I will take it out from `cea test` and instead update the docs on how to run `coverage` for the tests.